addons/rose/package.json

```diff
@@ -66,11 +66,12 @@
     "@storybook/cli": "^6.3.10",
     "@storybook/components": "^6.3.12",
     "@storybook/core-events": "^6.3.0",
-    "@storybook/ember": "^6.3.8",
+    "@storybook/ember": "6.4.0",
```

Comment:
Will we also continue upgrading to `6.5` and maybe `7.0`? IIRC the fix needed to get ember somewhat working with storybook was made in 7 but also eventually backported to `6.5`. 

I had tested a little while back after they had backported but still couldn't get it to work but with all these dependency fixes we might be there?

Reply:
Absolutley!! We plan on keep the upgrade going on, but first, after my experience working with so many dependency jungle is that we should move forward auditing resolutions, because that was in first place what unlocked this upgrade :smile:
I know it sounds cumbersome but I think is the smart way to work wich such depedency hell.